gui/main_window.py

```diff
@@ -1,29 +1,29 @@
-import sys
 import logging
 
-from PyQt5 import QtGui
-from PyQt5.QtWidgets import QAction, QLabel, QScrollArea, QMenu, QSizePolicy, QApplication, QMainWindow
+from dicom_app import dicom_reader
```

Comment:
Local application imports should be last

Reply:
Following PEP8:

Imports should be grouped in the following order:

    1. Standard library imports.
    2. Related third party imports.
    3. Local application/library specific imports.

You should put a blank line between each group of imports.